VirtoCommerce.Platform.Data.Security/Services/RefreshTokenServiceImpl.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Threading.Tasks;
+using VirtoCommerce.Platform.Core.Common;
+using VirtoCommerce.Platform.Core.Security;
+using VirtoCommerce.Platform.Data.Model;
+using VirtoCommerce.Platform.Data.Repositories;
+
+namespace VirtoCommerce.Platform.Data.Security.Services
+{
+    public class RefreshTokenServiceImpl : IRefreshTokenService
```

Comment:
Don't use `Impl` suffix. There is no such word.

Reply:
Got it, renamed to `RefreshTokenService`.